homeassistant/components/microbees/climate.py

```diff
@@ -0,0 +1,139 @@
+"""Climate integration microBees."""
+from typing import Any
+
+from microBeesPy import Sensor
+
+from homeassistant.components.climate import (
+    ClimateEntity,
+    ClimateEntityFeature,
+    HVACMode,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import ATTR_TEMPERATURE, UnitOfTemperature
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import MicroBeesUpdateCoordinator
+from .entity import MicroBeesActuatorEntity
+
+CLIMATE_PRODUCT_IDS = {76, 78}
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up the microBees climate platform."""
+    coordinator: MicroBeesUpdateCoordinator = hass.data[DOMAIN][
+        entry.entry_id
+    ].coordinator
+    async_add_entities(
+        MBClimate(
+            coordinator,
+            bee_id,
+            bee.actuators[0].id,
+            next(filter(lambda x: x.deviceID == 762, bee.sensors)).id
+            if bee.productID == 76
+            else next(filter(lambda x: x.deviceID == 782, bee.sensors)).id,
```

Comment:
Please avoid using multi-line if/else expressions.

Reply:
What do you mean? it is a ternary operator

Reply:
Exactly, avoid them being multi-line

Reply:
ruff format does it automatically

Reply:
Yeah, makes sense, ruff formats all our code.

Please restructure this to not have an multi-line ternary 👍